features/step_definitions/repository_steps.rb

```diff
@@ -113,6 +113,10 @@
   page.find('h3', text: text).click()
 end
 
+When(/^I click the "(.*?)" h3 unless I can see "(.*?)"$/) do |text, link|
```

Comment:
As far as I've understood the modification for the acceptance tests with "unless" was made because Analizo's accordion is already open. Can't we get this as a opportunity to make it start closed as all others? I believe this is a unwanted behaviour that is bothering us all.